content/docs/pipelines/upgrade.md

```diff
@@ -11,8 +11,13 @@ supports the following capabilities:
 * **Reinstall:** You can delete a cluster and create a new cluster, specifying
   the storage to retrieve the original data in the new cluster.
 
-Note that upgrade isn't currently supported, check [this issue](https://github.com/kubeflow/kubeflow/issues/3727)
-for progress.
+Note that upgrade isn't currently supported for Kubeflow deployment, check
+[this issue](https://github.com/kubeflow/kubeflow/issues/3727) for progress.
+
+We provide an alternative lite deployment to deploy Kubeflow Pipelines. It is
```

Comment:
s/lite/lightweight
(Unless branding decision says otherwise. See my comments in related PR.)